pgtap/withPoints/withPointsDD/edge_cases/issue_979.pg

```diff
@@ -2,7 +2,7 @@
 /*PGR-GNU*****************************************************************
 
 Copyright (c) 2018  pgRouting developers
-Mail: project@pgrouting.org
+Mail: project at pgrouting.org
```

Comment:
This test is failing when executed on an old version.

https://github.com/pgRouting/GSoC-pgRouting/actions/runs/5906112392/job/16021495975#step:9:176

I recommend the following:
Restart from scratch:

CREATE OR REPLACE FUNCTION test_old()
original code
DROP any table that was created
LANGUAGE plpgsql;

CREATE OR REPLACE FUNCTION test_new()
adjust original code to new signature
DROP any table that was created
LANGUAGE plpgsql;

CREATE OR REPLACE FUNCTION do_test()
IF min_version('3.6.0') then
RETURN QUERY  ... test_new_code()
ELSE
RETURN QUERY  ... test_old_code()
LANGUAGE plpgsql;
```